code/public/style.css

```diff
@@ -0,0 +1,58 @@
+:root {
+  background-color: #d2d6d9;
+}
+
+/*___________ New Task Input ___________*/
+
+.new-task-wrapper {
+  display: flex;
+  align-items: center;
+  margin: 20px 0 0 0;
+}
+
+.new-task-text {
+  margin: 8px;
+  flex-basis: 80%;
+}
+
+.new-task-submit {
+  display: flex;
+  justify-content: center;
+  margin: 0px 8px 0px 15px;
+}
+
+/*___________ Tasks ___________*/
+
+.task-list {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  margin: 10px 0px;
+}
+
+.task {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 6px 0px;
+  padding: 15px;
+  border-radius: 10px;
```

Comment:
https://www.w3schools.com/cssref/css3_pr_word-wrap.asp will save the day I think:D 
![image](https://user-images.githubusercontent.com/50926159/116782411-3288ce00-aa89-11eb-976b-ed7e1c93b393.png)